data/campaigns/Northern_Rebirth/scenarios/12a_Get_the_Gold.cfg

```diff
@@ -23,74 +23,81 @@
 
         [part]
-            story= _ "At Hidel’s call Tallin approached and knelt beside Hidel, bowing his head in respect."
+            story= _ "At Hidel’s call Tallin approached. He knelt beside, bowing his head in respect."
         [/part]
 
         {STORY_PART_TALLIN (_ "At your service, sir.")}
 
         {STORY_PART_HIDEL (_ "Eryssa has been... my charge ever... since she was born... I now return... to the earth... from which... I sprang... Please take care... of her.")}
 
-        {STORY_PART_TALLIN (_ "Rest assured, sir, I will place myself between Eryssa and any ill that strength or love may counter.")}
+        {STORY_PART_TALLIN (_ "I will.")}
 
         {STORY_PART_HIDEL (_ "Thank you... Tallin... Now I may rest... in... peace... May you... be victorious...")}
 
         [part]
-            story= _ "With these words Hidel closed his eyes and breathed his last. As Eryssa wept with Hidel’s head in her lap, Tallin approached her, hesitating, and then wrapped her gently in his arms."
+            story= _ "Then Hidel closed his eyes and took one last breath."
+        [/part]
+
+        [part]
+            story= _ "Slowly Tallin aproached her. He hesitated before wrapping her gently in his arms."
             music=love_theme.ogg
         [/part]
         [part]
-            story= _ "Sobbing against Tallin’s chest, Eryssa told him of the many ways Hidel had cherished and warded her since the hour of her birth. How he had comforted her childhood hurts; how in later years, he had guided her in the green paths of the forest, and taught her the deep mysteries of elven-kind."
+            story= _ "Sobbing against Tallin’s chest, Eryssa told him of the many ways Hidel had cherished and warded her and her sister. How he had comforted their childhood hurts; how in later years, he had guided them in the green paths of the forest, and taught them the deep mysteries of elven-kind."
         [/part]
         [part]
-            story= _ "A torrent of childhood memories and the depth of her grief left her unable to speak. She and Tallin sat together silently holding each-other for some time."
+            story= _ "The torrent of childhood memories and the depth of her grief left her unable to speak. She and Tallin sat together silently holding each-other for some time."
         [/part]
         [part]
-            story= _ "As Tallin comforted Eryssa, the rest of the party spread out and rallied the scattered elves, and tending to the wounded. This being done, they once again came before Tallin and Eryssa."
+            story= _ "As Tallin comforted Eryssa, the surviving elves gathered all the fallen comrades and with the human's help laid them to rest. For Hidel, the dwarves raised a monument so that his heroic death would long be remembered. Then they came before Tallin and Eryssa."
```

Comment:
```suggestion
            story= _ "As Tallin comforted Eryssa, the surviving elves gathered all of the fallen warriors, and with the humans’ help, laid them to rest. For Hidel, the dwarves raised a monument so that his heroic death would long be remembered. Then, they came before Tallin and Eryssa."
```